sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_shared/models.py

```diff
@@ -386,7 +386,7 @@ def from_string(cls, permission):
         :param str permission: Specify permissions in
             the string with the first letter of the word.
         :return: An AccountSasPermissions object
-        :rtype: ~azure.storage.filedatalake.AccountSasPermissions
+        :rtype: ~azure.storage.blob.AccountSasPermissions
```

Comment:
This should stay datalake in datalake. A few other instances of this in other files.